src/stdio_mgr/stdio_mgr.py

```diff
@@ -34,14 +34,19 @@
 
 
 class _PersistedBytesIO(BytesIO):
-    """Class to persist the stream after close.
+    """Class to persist the value after close.
 
-    A copy of the bytes value is available at :attr:`_closed_buf` after
+    A copy of the bytes value is given to a callback prior to
```

Comment:
Should probably document here in the docstring how a user should provide the callback?

The class itself is private, but someone bold may want to use it anyways.

Reply:
It is part of the mandatory args for the `__init__` , which are being documented by Sphinx ?

I dont see the value of documenting private classes in detail -- anyone going down this path should read the source.

Reply:
A private class can, and in this case, does and will continue to, change.

What good is helping people use it with good documentation, and then break it.